### ISSUE TITLE
Pretrained Models 

### ISSUE THREAD

**Issue by TiloccaS** [nesl/agrobot#3]
Hi I wanted to ask if the ATE and RTE present in the paper for the unseen trajectory of phase 1 referred to an average of the ate and rte obtained by the results of log8 and log5 or if they are the best results, which of the pretrained model should be used for phase 1?

Thank you

**Comment by swapnilsayansaha** [comment]
Run all (3) the pretrained models on phase 1 on the test set (for phase 1, log8 and log5), then take the average of the best value for log8 and log5.

**Comment by TiloccaS** [comment]
What's the difference between the 3 pretrained models?

**Comment by swapnilsayansaha** [comment]
Due to the stochastic nature of machine learning, you would typically report results by training a model multiple times from scratch. There is no difference architecturally or in the way they were trained among the 3 models per phase.
